src/main/java/com/chekanova/imagetool/service/comparison/impl/ImageComparisonServiceImpl.java

```diff
@@ -0,0 +1,193 @@
+package com.chekanova.imagetool.service.comparison.impl;
+
+import com.chekanova.imagetool.service.comparison.ImageComparisonService;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.awt.*;
```

Comment:
.* imports all classes from the package - import only exact classes from the package

Reply:
Done